napari/utils/_tests/test_migrations.py

```diff
@@ -26,3 +26,35 @@ def __init__(self, b) -> None:
     assert Sample(b=1).b == 1
     with pytest.deprecated_call():
         assert Sample(a=1).b == 1
+
+
+def test_deprecated_property() -> None:
+    class Dummy:
+        def __init__(self) -> None:
+            self._value = 0
+
+        @property
+        def new_property(self) -> int:
+            return self._value
+
+        @new_property.setter
+        def new_property(self, value: int) -> int:
+            self._value = value
+
+    instance = Dummy()
+
+    add_deprecated_property(
+        Dummy, "old_property", "new_property", "0.1.0", "0.0.0"
+    )
+
+    assert instance.new_property == 0
+
+    instance.new_property = 1
+
+    with pytest.warns(FutureWarning):
+        assert instance.old_property == 1
+
+    with pytest.warns(FutureWarning):
+        instance.old_property = 2
```

Comment:
`pytest.warns` has a match argument that allows to check if the proper warning is emitted. Could you check this?

Reply:
thanks @Czaki, adding this caught a bug.